src/test/java/org/dinosaur/foodbowl/domain/post/docs/PostControllerDocsTest.java

```diff
@@ -79,7 +79,7 @@ void findThumbnailsInProfile() throws Exception {
                 fieldWithPath("last").description("마지막 페이지 여부"),
                 fieldWithPath("hasNext").description("다음 페이지 존재 여부"),
                 fieldWithPath("currentPage").description("현재 페이지 인덱스"),
```

Comment:
```suggestion
                fieldWithPath("currentPageIndex").description("현재 페이지 인덱스"),
```
혹시 이 필드 `currentPageIndex`로 바꾸기에는 늦었을까요?! 🥺

Reply:
늦지 않았습니다! 확실히 의미가 인덱스라서 `Index`로 하는 것이 좋은 것 같습니다 꼼꼼한 리뷰 감사합니다 🙇🏻‍♂️👍